recipes/doubleml/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "doubleml" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/D/DoubleML/DoubleML-{{ version }}.tar.gz
+  sha256: 32ef1d20cd44cf628313a6a27c6ab00041414a531fb0a5bec3e7c77c2dfde875
+
+build:
+  number: 0
+  skip: true  # [py<36]
```

Comment:
Remove `skip` and make this `noarch: python` and we'll restrict the python version below:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python